.github/ISSUE_TEMPLATE/z_release_major.md

```diff
@@ -42,17 +40,14 @@ If all is good, then in 24h, I will start the release. Thanks!
   - [ ] Create a new document for the release in the [releases folder](https://github.com/medic/cht-docs/tree/main/content/en/core/releases).
   - [ ] Ensure all issues are in the GH Milestone, that they're correctly labelled (in particular: they have the right Type, "UI/UX" if they change the UI, and "Breaking change" if appropriate), and have human readable descriptions.
   - [ ] Use [this script](https://github.com/medic/cht-core/blob/master/scripts/release-notes) to export the issues into our release note format.
-  - [ ] Manually document any known migration steps and known issues.
-  - [ ] Provide description, screenshots, videos, and anything else to help communicate particularly important changes.
+  - [ ] Collect known migration steps, descriptions, screenshots, videos, data, and anything else to help communicate particularly important changes. This information should already be on the issue, but if not, prompt the change author to provide it.
   - [ ] Document any required or recommended upgrades to our other products (eg: cht-conf, cht-gateway, cht-android).
   - [ ] Add the release to the [Supported versions](https://docs.communityhealthtoolkit.org/core/releases/#supported-versions) and update the EOL date and status of previous releases. Also add a link in the `Release Notes` section to the new release page.
```

Comment:
How does the status work for that table? I added the release that I was doing as `Current`, the previous one as `Supported`, and the one before as `EOL` but I am not 100% sure about that, could there be more than one version set as `Supported` for `4.x.x` for example? Can we specify that in the step please?

Reply:
Clarified.